enclave/core/switchlesscalls.c

```diff
@@ -105,6 +106,18 @@ oe_result_t oe_post_switchless_ocall(oe_call_host_function_args_t* args)
                     NULL,
                     args))
             {
+                // Set the worker's state to RUNNING.
+                if (__sync_val_compare_and_swap(
+                        &_host_worker_contexts[tries].event, 0, 1) == 0)
```

Comment:
Related to the previous code: Why do we iterate backwards rather than forwards?

Reply:
There is no technical reason. Just seems to be a coding style. Keeping it as is.